doc/source/conf.py

```diff
@@ -157,6 +160,98 @@
     }
 
 # -- Jinja context configuration ---------------------------------------------
+
+def zip_directory(directory_path: pathlib.Path, zip_filename: pathlib.Path, ignore_patterns=None):
+    """Compress a directory using ZIP.
```

Comment:
```suggestion
    """Compress a directory using the Zip app.
```
Based on their product documentation, the app name is Zip (only initial capital letter). A reference to the file extension would be in all capital letters, such as "Download the ZIP file."